src/global/global.scss

```diff
@@ -0,0 +1,5 @@
+* {
```

Comment:
Move `src/global/global.scss` to a folder like `style` or `scss` for better maintenance